data/blog/tclip.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: 'tclip: A pastebin for your tailnet'
+date: 2023-08-11
+summary: Sometimes you need to paste text between your machines or coworkers. tclip is a Tailscale-powered pastebin that lets you do just that.
+authors: ['xe']
+tags:
+  - tsnet
+  - funnel
+images: ['/images/tclip/social.jpg']
+---
+
+As a developer, there are many times when you need to share little snippets of text with your friends or team. The tool I used the most for this is GitHub [gists](https://gist.github.com). This lets you upload little snippets of text, and it a publicly shareable link to it.
+
+While I was playing around with tsnet, I got the idea to make a little pastebin clone using it. And then Funnel was announced. The hacker gear started turning, and I did a little exploration of the problem space and came up with tclip. I wanted to make a tool that I would feel comfortable with using every day, if I got to bring in a bunch of other cool technology into it as well, that would be even more cool.
+
+So I ended up with tclip.
+
+<Image
+  className="mt-2"
+  src="/images/tclip/frontpage.png"
+  width={1804}
+  height={870}
+  alt="The homescreen of tea clip, showing a textbox labeled 'create new paste'"
+/>
+
+The core of the entire program can be summarized in this entire image. It is a box that you can type text in, and submit it to the server. Then you get a link that you can use to share with your friends and coworkers. That's it. That's the entire program.
+
+If you put code in the box and then name the file appropriately, your file will be syntax highlighted. There's a commandline tool that you can use to pipe the output of commands or upload files directly to your tclip server so you can share them with your team and friends.
+
+If you click on the `List` link in the top, you can see a list of all the pastes that have been created recently, but the main screen will show the last 5 pastes just so you can get to the one you want to see a little bit quicker. If you put [markdown](https://en.wikipedia.org/wiki/Markdown) in the box and end your filename in `.md`, it'll be rendered as HTML for you. We've hidden a few other things to spark joy like this, like an [Emacs package](https://github.com/tailscale-dev/tclip/blob/main/contrib/tclip.el) and the ability to run as a [systemd portable service](https://systemd.io/PORTABLE_SERVICES/).
+
+<HeroImage
+  width={1280}
+  height={512}
+  desc="An albino pidgeon behind a normal one as found on Oberbaumbrücke in Berlin. Photo by Xe Iaso, 2023."
+  src="/images/tclip/hero.jpg"
+/>
+
+## Installation
+
+The easiest way to use tclip is and to install it on fly.io. You can do that by doing the following:
+
+First, get an authkey from [the admin panel](https://login.tailscale.com/admin/settings/keys). It is a good idea to associate this with the tag `tag:service` or its own `tag:tclip`. Please also do not set the `ephemeral` tag as that will destroy the node when the service shuts down.
```

Comment:
We don't use "please" in the KB (go/wordlist). 

Not sure if the same applies to the blog post. "Please" is also mentioned on the last line in this file.